connector-routing-impl/src/main/java/org/interledger/connector/routing/InterledgerAddressPrefixMap.java

```diff
@@ -211,6 +215,18 @@ public void forEach(final BiConsumer<? super InterledgerAddressPrefix, R> action
     }
   }
 
+  /**
+   * Filters entries from {@code toFilter} where entry.getKey() is longer than the prefixKey.
```

Comment:
Let's add a concrete example here for posterity: 

```suggestion
   * Filters entries from {@code toFilter} where entry.getKey() is longer than the prefixKey. For example, if the Trie has an entry for `g.foo` and `g.foo.bar.baz`, then when looking for a match for a destination address of `g.foo.bar`, the trie will choose `g.foo.bar.baz`. While correct behavior for a Trie, this is not what the ILP longest-prefix-match desires. To rectify this, we need to exclude any results from the trie that are longer than the destination address being matched upon.
```